meetings/agenda-minutes-2019-01-08.md

```diff
@@ -80,7 +80,7 @@ Agenda and notes  (please suggest your own additions/refinements):
         * RG: I'd like this to be a sequence of proposals.
         * JW: I could believe that proceeding in stages could be useful, if we're defining one bit of syntax, continuing on, we could collect in-field telemetry.
         * RG: It's also possible that we could end up in a good state where the spec says, if the input is not fully specified, then implementations are free to fall back to whatever, but it's recommended to take a certain action. When I talk about multiple proposals, I'm talking about two: the preferred syntax, and then defining it fully. I don't want to do nothing just because we can't do everything.
-        * DE: Maybe we could start towards trying to define things completely, and then 
+        * DE: Maybe we could start towards trying to define things completely, and then
```

Comment:
I try not to touch whitespace like this, but there's some formatter that's being insistent about it, so I included the change. Happy to drop.